common-util/src/main/java/org/sunbird/common/models/util/EmailValidator.java

```diff
@@ -0,0 +1,33 @@
+package org.sunbird.common.models.util;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.apache.commons.lang.StringUtils;
+
+public class EmailValidator {
+
+  private static Pattern pattern;
+  private static final String EMAIL_PATTERN =
+      "^[_A-Za-z0-9-\\+]+(\\.[_A-Za-z0-9-]+)*@"
+          + "[A-Za-z0-9-]+(\\.[A-Za-z0-9]+)*(\\.[A-Za-z]{2,})$";
+
+  private EmailValidator() {}
+
+  static {
+    pattern = Pattern.compile(EMAIL_PATTERN);
+  }
+
+  /**
+   * Method to validate email.
+   *
+   * @param email Email value
+   * @return boolean Is email valid or not.
+   */
+  public static boolean isEmailvalid(String email) {
```

Comment:
Validates format of email.

email Email value
True, if email format is valid. Otherwise, return false.

Rename to isEmailValid.